src/reference/asciidoc/whats-new.adoc

```diff
@@ -68,3 +68,9 @@ See <<./amqp.adoc#amqp-inbound-channel-adapter,AMQP Inbound Channel Adapter>>
 The `encodeUri` property on the `AbstractHttpRequestExecutingMessageHandler` has been deprecated in favor of newly introduced `encodingMode`.
 See `DefaultUriBuilderFactory.EncodingMode` JavaDocs and <<./http.adoc#http-uri-encoding,Controlling URI Encoding>> for more information.
 This also affects `WebFluxRequestExecutingMessageHandler`, respective Java DSL and XML configuration.
+
+[[x5.3-WebFlux]]
+=== WebFlux Changes
+
+The `timeoutFunction` property is added to `WebFluxRequestExecutingMessageHandler`.
+Java DSL accessor which accepts long Timeout, Duration Timeout & Timeout Function is provided
```

Comment:
The good sentence ends with period. 
I don't think that word "timeout" deserves to be capitalized. 
I would even say if your don't talk about code, then even `function` word doesn't deserve to be capitalized. 

Would be great to have here a brief description for the reason of this new option. See something already existed in this doc.
Then we definitely need a link to the target docs.
And what is very important we need a description in the final docs - `webflux.adoc`.